dora/data_inclusion/constants.py

```diff
@@ -0,0 +1,17 @@
+# À une thématique DI correspond une thématique Dora
+THEMATIQUES_MAPPING_DI_TO_DORA = {
+    "logement-hebergement--etre-accompagne-dans-son-projet-accession": "logement-hebergement--etre-accompagne-pour-se-loger",
+    "logement-hebergement--etre-accompagne-en cas-de-difficultes-financieres": "logement-hebergement--gerer-son-budget",
+    "logement-hebergement--financer-son-projet-travaux": "logement-hebergement--autre",
+}
+
+# Inversion du dictionnaire
+# À une thématique Dora correspond une liste de thématiques DI
+THEMATIQUES_MAPPING_DORA_TO_DI = {}
+for key, value in THEMATIQUES_MAPPING_DI_TO_DORA.items():
+    if value.endswith("--autre"):
+        continue
+    if value not in THEMATIQUES_MAPPING_DORA_TO_DI:
+        THEMATIQUES_MAPPING_DORA_TO_DI[value] = [key]
+    else:
+        THEMATIQUES_MAPPING_DORA_TO_DI[value].append(key)
```

Comment:
suggestion: utiliser la méthode _built-in_ [`collections.defaultdic`](https://docs.python.org/3/library/collections.html#collections.defaultdict), qui assure de n'avoir pas 2 fois la même clé dans les éléments de la liste, pour plus de concision / lisibilité (et un peu perf) :

```suggestion
from collections import defaultdict

THEMATIQUES_MAPPING_DORA_TO_DI = defaultdict(list)
for key, value in THEMATIQUES_MAPPING_DI_TO_DORA.items():
    if not value.endswith("--autre"):
        THEMATIQUES_MAPPING_DORA_TO_DI[value].append(key)
```

Reply:
Bonne idée le `defaultdict`. 
Faire simplement attention si on accepte une liste vide comme valeur valide pour le champ.

Reply:
Je ne connaissais pas `defaultdict`. C'est effectivement beaucoup mieux avec ! Merci :)

Modification faite : 5c4d39367e22923d544712b6ebbaf3332a29b09d